src/index.js

```diff
@@ -19,13 +48,15 @@ const getApplicationId = (token: string): string | null => {
     return null;
   }
 };
-
-const appDir: string = path.join(__dirname, "../app");
-const appPackage: Package = require(path.join(appDir, "package.json"));
-const { name, version }: Package = require(path.join(
-  __dirname,
-  "../package.json"
-));
+/**
+ * @type {string}
+ */
+const appDir = path.join(__dirname, "../app");
+/**
+ * @type {Package}
+ */
+const appPackage = require(path.join(appDir, "package.json"));
+const { name, version } = require(path.join(__dirname, "../package.json"));
```

Comment:
can we add `@type {Package}` docs for assignment here? or are there issues when destructure assigning?

Reply:
Yeah, JSDoc with Typescript still has some limitations and stuff like destructuring is one of them.